astropy/nddata/ccddata.py

```diff
@@ -129,6 +129,11 @@ class CCDData(NDDataArray):
             If the unit is ``None`` or not otherwise specified it will raise a
             ``ValueError``
 
+    psf : `numpy.ndarray` or None, optional
```

Comment:
I believe a psf image should always be unitless, but I'm not positive of that.

Reply:
I am just thinking what if someone stored it as data with pixel unit attached to it.

Reply:
hmmm... I'm not sure what to do on that point: I think the psf image as being purely an array (because the thing you do with it is (de)convolve it with the image), and I can't think of a reason someone would want it to have pixel units or what would be gained by that. Might be worth waiting to see how use-cases shake out and if anyone requests it?

Reply:
That is fine. I was just curious if this was discussed.